.gitignore

```diff
@@ -0,0 +1,205 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
yes. since the binplaced location contains built-binaries and the user does not need to push to github